components/LinkedDataProof.yml

```
@@ -22,6 +22,15 @@ components:
         created:
           type: string
           description: Date the proof was created.
+        challenge:
+          type: string
+          description: A challenge to mitigate replay attacks.
+        domain:
+          type: string
+          description: The domain of the proof to restrict its use to a particular target.
+        nonce:
+          type: string
+          description: A nonce to mitigate replay attacks.
```

```suggestion
          description: A value chosen by the creator of a proof to randomize proof values for privacy purposes.
```

A `nonce` can also be used to mitigate replay attacks but `challenge` is a better for that.